docs/installation.rst

```diff
@@ -14,26 +14,38 @@ Installing ``openfe``
 When you install ``openfe`` through any of the methods described below, you
 will install both the core library and the command line interface (CLI). 
 
-Installation with ``conda`` (recommended)
+Installation with ``micromamba`` (recommended)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-We recommend installing ``openfe`` with ``conda``, because it provides easy
+We recommend installing ``openfe`` with ```micromamba`` <https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html#micromamba>`_ , because it provides easy
 installation of other tools, including molecular dynamics tools such as
-OpenMM and ambertools, which are needed by ``openfe``. 
-Conda can be installed either with the `full Anaconda distribution
-<https://www.anaconda.com/products/individual>`_, or with
-the `smaller-footprint miniconda
-<https://docs.conda.io/en/latest/miniconda.html>`_.
+OpenMM and ambertools, which are needed by ``openfe``.
 
-You can get ``openfe`` from the ``conda-forge`` channel. To install the most
-recent release in its own environment, use the commands ::
+To install ``micromamba`` execute the `installation script <https://mamba.readthedocs.io/en/latest/installation.html#install-script>`_.
 
-  $ conda create -n openfe -c conda-forge openfe
-  $ conda activate openfe
+On Linux the default shell is ``bash`` ::
+  
+  $ curl micro.mamba.pm/install.sh | bash
+
+And on macOS the default shell is ``zsh`` ::
+
+  $ curl micro.mamba.pm/install.sh | zsh
+
+Now we will set ``conda-forge`` as the default channel ::
+
+  $ micromamba config append channels conda-forge
+
+To install ``openfe`` run ::
```

Comment:
"This creates an environment called `openfe_env` which contains the `openfe` package, and all dependencies.

Reply:
Done!